completions/ccache

```diff
@@ -5,12 +5,15 @@ _ccache()
     local cur prev words cword split
     _init_completion -s || return
 
-    if [[ $COMP_CWORD -eq 1 && ${COMP_WORDS[COMP_CWORD]} != -* ]]; then
-        _command_offset 1
-        return
-    fi
+    for (( i=1; i <= COMP_CWORD; i++ )); do
+        if [[ ${COMP_WORDS[i]} != -* ]]; then
+            _command_offset $i
+            return
+        fi
+        [[ ${COMP_WORDS[i]} == -@(o) ]] && ((i++))
+    done
 
-    case $prev in
+    case "$prev" in
```

Comment:
This looked wrong to me, but as you think it's cosmetic (i.e. prev cannot be empty), I've removed it.